lanl_cice/drivers/cice/CICE_InitMod.F90

```diff
@@ -187,9 +189,7 @@ subroutine init_restart
           restart_pond_topo, read_restart_pond_topo
       use ice_restart_shared, only: runtype, restart
       use ice_restart_driver, only: restartfile, restartfile_v4
-      use ice_state, only: tr_iage, tr_FY, tr_lvl, tr_pond_cesm, &
-          tr_pond_lvl, tr_pond_topo, tr_aero, trcrn, &
-          nt_iage, nt_FY, nt_alvl, nt_vlvl, nt_apnd, nt_hpnd, nt_ipnd, tr_brine
+      use ice_state ! almost everything
```

Comment:
I think it is good to list all the variables in the ice_state as it improves readability (you can see all the variables and where they are defined).